projects/storefrontlib/src/cms-components/configurator/generic/service/config-router-extractor.service.ts

```diff
@@ -68,12 +72,16 @@ export class ConfigRouterExtractorService {
     return owner;
   }
 
-  getConfiguratorTypeFromUrl(url: string): string {
+  protected getConfiguratorTypeFromSemanticRoute(route: string): string {
```

Comment:
Since we pass `semanticRoute` instead of `route` as an argument perhaps we should change a little the behavior of the method:
- I'm not sure splitting with `/` still makes sense 
- route.includes probably can be changed to strict equality === instead
- bagatelle: optionally we can rename the funciton argument from `route` to `semanticRoute`

Reply:
We still need include, as the routes can be configureTEXTFIELD, configureCPQCONFIGURATOR (which stands for VC, historic reasons) or configureCLOUDCPQ (later, for CPQ). The methods returns the configurator type according to the commerce

Reply:
Yes, will rename function argument

Reply:
Thanks, now I can see it's a _fragment_!

Do you think that the following could be optimized? :
- I believe the `semanticRoute` doesn't contain slashes, so we could drop logic `split('/')`.
- `.includes()` with could be replaced with `.startsWith()`, to be more precise
- It would be nice to add a JSDoc comment explaining the context: the semantic routes for various configurators consist of 2 parts: common prefix `configure` (or `configureOverview`) and the dynamic configurator type as a suffix. And in this method we extract the second part.

Reply:
Yes, agree. I will do this in my next PR that will contain the renaming activities